docs/user.md

```diff
@@ -688,6 +688,25 @@ manifest the operator will raise the limits to the configured minimum values.
 If no resources are defined in the manifest they will be obtained from the
 configured [default requests](reference/operator_parameters.md#kubernetes-resource-requests).
 
+### HugePages support
```

Comment:
Seems this comes with K8s 1.28 

Can we also link https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/

Reply:
Done